IHP/QueryBuilder.hs

```diff
@@ -377,6 +389,22 @@ filterWhereNotIn (_, []) = id -- Handle empty case by ignoring query part: `WHER
 filterWhereNotIn (name, value) = FilterByQueryBuilder (name, NotInOp, toField (In value))
 {-# INLINE filterWhereNotIn #-}
 
+filterWhereLike :: forall name table model value. (KnownSymbol name, ToField value, HasField name model value, model ~ GetModelByTableName table) => (Proxy name, value) -> QueryBuilder table -> QueryBuilder table
```

Comment:
Can we add haddock blocks to all these new functins, like this?

```haskell
-- | Adds a simple @WHERE x LIKE y@ condition to the query.
--
-- __Example:__ (we need to think of a real world like example here)
--
-- > activeProjects <- query @Project
-- >     |> filterWhere (#active, True)
-- >     |> fetch
-- > -- SELECT * FROM projects WHERE active = True
```

Those show up autocompletion when using haskell language server, so it's kind of useful :)